src/components/Details/VmDetails/fixtures/VmDetails.fixture.js

```diff
@@ -113,20 +114,19 @@ const vmiFixture = {
   },
 };
 
+const k8sGet = () =>
```

Comment:
we could add this to tests/k8sGet function 
note: k8sGet should be converted to return promise

Reply:
fixed